executable/swf.ksy

```diff
@@ -90,6 +96,48 @@ types:
         type: u1
       - id: b
         type: u1
+  # Swf specs has 3 types for color, rgb, rgba and arg.
```

Comment:
This comment doesn't bring much clarity.

Reply:
True that. Happy to remove it. I think It's not obvious how to explain a 215 page specs in a few comments 😓